File: core/src/com/mygdx/game/characters/State/WalkingRight.java
package com.mygdx.game.characters.State;

import com.mygdx.game.characters.GameCharacter;

public class WalkingRight extends State {
    public WalkingRight(GameCharacter character) {
        super(character);
    }

    public void stand() {
        character.state = new Stand(character);
    }
    public void runLeft() {}
    public void runRight() {}
    public void walkLeft() {}
    public void walkRight() {}

    @Override
    public void climbUp() {

    }

    @Override
    public void climbDown() {

    }
}

File: web/src/characters/state/WalkingRight.ts
import { State } from './State';
import { Stand } from './Stand';

export class WalkingRight extends State {
  stand(): void {
    this.character.fsm = new Stand(this.character);
  }
  runLeft(): void {}
  runRight(): void {}
  walkLeft(): void {}
  walkRight(): void {}
  climbUp(): void {}
  climbDown(): void {}
}
